docs/src/reference/attributes.md

```diff
@@ -0,0 +1,232 @@
+# Attributes
```

Comment:
Suggest that you link to this page (or the right section) from the tutorial, when introducing each attribute, so that users can learn more?

Reply:
Done! Only `#[kani::proof]` and `#[kani::unwind(<number>)]` are used in the tutorial, so I've added links to those items when they're first mentioned.